src/assets/source/sass/components/_navbar.scss

```diff
@@ -1,10 +1,74 @@
 .navbar {
-  i {
-    color: white;
-  }
+  padding: 20px;
 }
+
 .dropdown {
   i {
+    margin-right: 5px;
     color: black;
   }
 }
+
+.form-inline {
+  margin-left: 67%;
+
+  #__BVID__9 {
```

Comment:
it looks like this is an auto generated ID, it might change in the future, so it is not a good practice to use it as a selector